### PR TITLE
HomeAppTV: add service card for video

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1447
[Problem] Implement webclip for video
[Solution]
 - new webclip has been added

[todo]
test fail:
tests/js/core/widget/core/SectionChanger/SectionChanger2.html

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
I tested this PR on Android mobile (https://lslachciak.github.io/TAU/examples/mobile/HomeApp/index.html)
Issues:
- two last videos (Ch4) not visible at all
- first video area  is scrollable up and down
image

**Comment by lslachciak** [comment]
in case of tablet video is bigger than widget area
but this can be another issue, not related with this  PR
image

**Comment by TomaszLukawskiSam** [comment]
> I tested this PR on Android mobile (https://lslachciak.github.io/TAU/examples/mobile/HomeApp/index.html)
> Issues:
> 
> * two last videos (Ch4) not visible at all
> * first video area  is scrollable up and down
>   image

I'll change video:
https://download.blender.org/peach/bigbuckbunny_movies/big_buck_bunny_480p_surround-fix.avi

**Comment by singa2000** [comment]
@TomaszLukawskiSam 
Would you fix the CI build error and prepare the PR, please?

**Comment by TomaszLukawskiSam** [comment]
> @TomaszLukawskiSam
> Would you fix the CI build error and prepare the PR, please?

Ok. I'll fix it ASSAP.

File: examples/mobile/HomeApp/webclip/video-service/webclip.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+					</div>
+					<div class="ui-footer">
+						<button class="ui-btn-contained-colored" data-inline="true" onclick="playOnTV(3,0)">Play On TV</button>
+					</div>
+				</section>
+
+				<section class="ui-content-area">
+					<div class="ui-content">
+						<video muted autoplay src="https://file-examples-com.github.io/uploads/2017/04/file_example_MP4_480_1_5MG.mp4" loop
+							id="video4" style="width: 100%" onclick="this.play()"></video>
```
Comment (singa2000) [inline examples/mobile/HomeApp/webclip/video-service/webclip.html:49]: It's same video with "Ch4 : Toy World". Would you change the video like "http://clips.vorwaerts-gmbh.de/VfE_html5.mp4" , and title is "Ch1: BigBuckBunny" ,please?

File: examples/mobile/HomeAppServer/app.js [hunk 2]
```diff
@@ -95,6 +95,17 @@ var express = require("express"),
...
+	{
+		"appID": "vUf39tzQvi.Video",
+		"isInstalled": false,
+		"isActive": false,
+		"webClipsList": [
+			{
+				url: "webclip/video-service",
+				isSelected: true
+			}
+		]
```
Comment (singa2000) [inline examples/mobile/HomeAppServer/app.js]: Is to also need the "action":"add"?

File: examples/mobile/HomeApp/webclip/video-service/webclip.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<div class="ui-card ui-card-service" class="app-video-control">
+	<link rel="stylesheet" href="css/style.css" />
+	<div class="ui-content-subheader">
+		Video Collection
+	</div>
+	<div class="ui-content-area">
+		<div class="ui-section-changer">
+			<div class="ui-section-changer-container">
+				<section>
```
Comment (singa2000) [inline examples/mobile/HomeApp/webclip/video-service/webclip.html]: Is it needed  class="ui-content-area" like <section class="ui-content-area">?

File: examples/mobile/HomeAppServer/app.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeAppServer/app.js]: You are right. I'll add "action: add".

File: examples/mobile/HomeApp/webclip/video-service/webclip.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/webclip.html:49]: I'll change.

There will be potential issue with "http" protocol if home app will be open with "https". This source is available only by "http".

File: examples/mobile/HomeApp/webclip/video-service/webclip.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/webclip.html]: You are right.

File: examples/mobile/HomeAppServer/app.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeAppServer/app.js]: Done.

File: examples/mobile/HomeApp/webclip/video-service/webclip.html (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/webclip.html:49]: Done.

File: examples/mobile/HomeApp/webclip/video-service/webclip.html (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/webclip.html]: Done.

File: examples/mobile/HomeApp/webclip/video-service/css/style.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+.app-control-status .app-control-video-subtitle {
+    font-size: 16px;
+    color: var(--text-secondary-color);
+}
+
+.ui-section-changer {
+	height: 100% !important;
+}
+.ui-section-changer-container {
```
Comment (lslachciak) [inline examples/mobile/HomeApp/webclip/video-service/css/style.css:32]: ui-section-changer-container was added to SectionChanger.js 
so shouldn't we add CSS also to TAU library?

Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/css/style.css:32]: At the moment this cause issues with tests. These styles will be specific for this app.

File: examples/mobile/HomeApp/webclip/video-service/webclip.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+					<div class="ui-footer">
+						<button class="ui-btn-contained-colored" data-inline="true" onclick="playOnTV(3,0)">Play On TV</button>
+					</div>
+				</section>
+
+				<section class="ui-content-area">
+					<div class="ui-content">
+						<video muted autoplay src="http://clips.vorwaerts-gmbh.de/VfE_html5.mp4" loop
+							id="video4" style="width: 100%" onclick="this.play()"></video>
+						<div class="ui-title">Ch1: BigBuckBunny</div>
```
Comment (lslachciak) [inline examples/mobile/HomeApp/webclip/video-service/webclip.html]: Ch1 should be first on list not last
It seems strange when testing app:
Ch4...Ch2..Ch3..Ch..1

**Review by lslachciak — COMMENTED** [review]
I have some comments in code.
I also tested PR, work generally OK. The only problem I see is that Ch3 video is smaller than others(due to aspect ratio?) so it have large empty space on the bottom. But I understand that reason of this is that other videos are larger.
106 120 63 110_8080_examples_mobile_HomeApp_(Galaxy S5) (3)

File: examples/mobile/HomeApp/webclip/video-service/webclip.html (hunk 5 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/HomeApp/webclip/video-service/webclip.html]: Fixed.

**Review by lslachciak — APPROVED** [review]

**Comment by TomaszLukawskiSam** [comment]
> I have some comments in code.
> I also tested PR, work generally OK. The only problem I see is that Ch3 video is smaller than others(due to aspect ratio?) so it have large empty space on the bottom. But I understand that reason of this is that other videos are larger.
> 106 120 63 110_8080_examples_mobile_HomeApp_(Galaxy S5) (3)

We are not ready for fix it yet.

**Pull request merged**